activesupport/lib/active_support/railtie.rb

```diff
@@ -13,6 +13,13 @@ class Railtie < Rails::Railtie # :nodoc:
       end
     end
 
+    initializer "active_support.halt_callback_chains_on_return_false", after: :load_config_initializers do |app|
```

Comment:
@claudiob do you remember why we are running this after `load_config_initializers`? This is making `load_config_initializers` run way early in the initializers order.